tests/nnx/summary_test.py

```diff
@@ -41,9 +41,10 @@ def __call__(self, x):
 
     foo = Foo(nnx.Rngs(0))
     x = jnp.ones((1, 32))
-    table_repr = nnx.tabulate(
+    table_repr_ = nnx.tabulate(
```

Comment:
Added because whenever something goes wrong, we have to print the table to debug. And having the repr on a separate line makes it easier to print the table later.